app/models/user.rb

```diff
@@ -1,24 +1,39 @@
 class User < ApplicationRecord
-  has_secure_password
-
+  attr_accessor :remember_token
   before_save :downcase_email
 
   validates :name, presence: true
   VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
   validates :email, presence: true, length: {maximum: 255}, format: {with: VALID_EMAIL_REGEX}, uniqueness: true
 
+  def remember
+    self.remember_token = User.new_token
+    update_attribute :remember_digest, User.digest(remember_token)
+  end
+
+  has_secure_password
```

Comment:
move this line place after validate scope
```
validates :email, presence: true, length: {maximum: 255}, format: {with: VALID_EMAIL_REGEX}, uniqueness: true
has_secure_password

# something method
```

Reply:
You need to fix comment